biorbd_optim/path_conditions.py

```diff
@@ -111,7 +111,7 @@ def __init__(self, biorbd_model, all_generalized_mapping=None, q_mapping=None, q
 
 
 class InitialConditions(PathCondition):
-    def __init__(self, initial_guess=()):
+    def __init__(self, initial_guess=(), init_lineaire = False):
```

Comment:
Make sure your variable names are in english

Reply:
Thank you for your answer. It was a mistake on my side, sorry